eogrow/core/area/base.py

```diff
@@ -77,10 +77,10 @@ def get_grid(self, filtered: bool = True) -> Dict[CRS, gpd.GeoDataFrame]:
         grid = self._create_grid()
         self._save_grid(grid, grid_path)
 
-        if filtered and self.config.patch_list is not None:
-            folder_path = self.storage.get_folder(self.config.patch_list.input_folder_key)
-            patch_list_path = fs.path.join(folder_path, self.config.patch_list.filename)
-            relevant_patches = set(load_eopatch_names(self.storage.filesystem, patch_list_path))
+        if filtered and self.config.patch_names_list is not None:
+            folder_path = self.storage.get_folder(self.config.patch_names_list.input_folder_key)
+            patch_names_path = fs.path.join(folder_path, self.config.patch_names_list.filename)
+            relevant_patches = set(load_eopatch_names(self.storage.filesystem, patch_names_path))
```

Comment:
`relevant_patches` should be more in line with `patch_names`/`patch_names_list`, because it's also just a list of names, right?